Document/0x04g-Testing-Cryptography.md

```diff
@@ -109,13 +101,6 @@ Periodically ensure that the cryptography has not become obsolete. Some older al
 - CWE-326: Inadequate Encryption Strength
 - CWE-327: Use of a Broken or Risky Cryptographic Algorithm
 
-##### Info
-- [1] Commercial National Security Algorithm Suite and Quantum Computing FAQ - https://cryptome.org/2016/01/CNSA-Suite-and-Quantum-Computing-FAQ.pdf
-- [2] NIST Special Publication 800-57 - http://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-57pt1r4.pdf
-- [4] NIST recommendations (2016) - https://www.keylength.com/en/4/
-- [5] BSI recommendations (2017) - https://www.keylength.com/en/8/
-- [6] Sweet32 attack -- https://sweet32.info/
-
 ##### Tools
 - QARK - https://github.com/linkedin/qark
 - Mobile Security Framework - https://github.com/ajinabraham/Mobile-Security-Framework-MobSF
```

Comment:
Good spot @Sjord. In order to stay consistent this should also be in the same style as the reference links.